File: data/python/PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES_MAIN.py
if __name__ == '__main__':
    param = [
    (11,),
    (27,),
    (31,),
    (47,),
    (3,),
    (14,),
    (41,),
    (72,),
    (39,),
    (22,)
        ]
    n_success = 0
    print("class,mutant,test_index,result")
    for i, parameters_set in enumerate(param):
        try:
            print("PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES," + sys.argv[1] + "," + str(i) + "," + str(f_gold(*parameters_set)))
        except:
            print("PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES," + sys.argv[1] + "," + str(i) + ",EXCEPTION")

File: data/java/PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES_MAIN.java
public static void main(String args[]) throws IOException {
    int n_success = 0;
    List<Integer> param0 = new ArrayList<>();
    param0.add(11);
    param0.add(27);
    param0.add(31);
    param0.add(47);
    param0.add(3);
    param0.add(14);
    param0.add(41);
    param0.add(72);
    param0.add(39);
    param0.add(22);
    StringBuilder builder = new StringBuilder();
    builder.append("class,mutant,test_index,result\n");
    FileWriter writer = new FileWriter(args[0]);
    for(int i = 0; i < param0.size(); ++i)
    {
        try {
            builder.append("PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES," + args[1] + "," + i + "," + f_gold(param0.get(i)) + "\n");
        } catch (Exception e) {
            builder.append("PRIMALITY_TEST_SET_5USING_LUCAS_LEHMER_SERIES," + args[1] + "," + i + ",EXCEPTION\n");
        }
    }
    writer.write(builder.toString());
    writer.close();
}
